examples/etcd/certs/Makefile

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
This is done to remove friction to adoption. `cfssl` is new and obscure and needs to be installed and learned. `openssl` is everywhere and typing `make` delivers "it just works out of the box" experience.